testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2442,7 +2442,7 @@ private List<SetColumnTypeSetup> setColumnTypeSetupData()
                 .build();
     }
 
-    public record SetColumnTypeSetup(String sourceColumnType, String sourceValueLiteral, String newColumnType, String newValueLiteral, boolean unsupportedType)
+    public record SetColumnTypeSetup(String sourceColumnType, String sourceValueLiteral, String newColumnType, String newValueLiteral, boolean unsupportedType, Optional<String> tableProperty)
```

Comment:
`Optional<String> tableProperty` -> `String tableProperties` 
(empty string means "no properties", no need to wrap in optional)